src/services/aws/bankDetails/bankDetails.ts

```diff
@@ -0,0 +1,19 @@
+import { WiseRequest } from "types/aws";
+import { aws } from "services/aws/aws";
+import { version as v } from "../../helpers";
+
+export const bank_details_api = aws.injectEndpoints({
+  endpoints: (builder) => ({
+    /**
+     * The endpoint returns different types of results depending on the requests sent, but since it's impossible
+     * to specify a generic return argument here, we return `unknown` and cast the return value manually
```

Comment:
For the casting logic, see [useTypedWiseMutation](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2439/files#diff-ffd7f4b3dc2dac7b058b975c84453fb7bb7badead212d7afa60b65884f6f4c88R18)

Reply:
This is outdated logic